network-store-server/src/main/java/com/powsybl/network/store/server/TemporaryLimitSqlData.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.server;
+
+import com.powsybl.iidm.network.LimitType;
+import com.powsybl.network.store.model.TemporaryLimitAttributes;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ */
+@ToString
+@Builder
+@AllArgsConstructor
+@Getter
+public class TemporaryLimitSqlData {
```

Comment:
Move to dto package or a new package.
And why do you need this class ?

Reply:
for json serialization